SeQuant/domain/mbpt/space_qns.hpp

```diff
@@ -49,11 +49,13 @@ struct mask<Spin> {
 };
 
 /// quantum numbers tags related to LCAO basis traits
-/// \note sLCAO basis traits use 3rd and 4th rightmost bits
+/// \note LCAO basis traits use 3rd and 4th rightmost bits
 enum class LCAOQNS : bitset_t {
-  ao = 0b000100,
-  pao = 0b001000,  // projected AO space denotes unoccupied spaces made from AO
-                   // basis and orthogonal to occupied orbitals
+  ao = 0b000100 | bitset_t(Spin::any),
+  pao =
+      0b001000 |
+      bitset_t(Spin::any),  // projected AO space denotes unoccupied spaces made
```

Comment:
Why are quantum numbers mixed like this? It seems like now it is impossible to set LCAO QNs without also setting spin QNs to `Spin::any` 🤔